Content/Maatregelen/M19/ICTU.md

```diff
@@ -1,3 +1,7 @@
 #### ICTU
 
 ICTU ondersteunt dit met Docker en/of virtuele machines (VM) en een VLAN per project. Een nieuwe afgeschermde digitale omgeving is binnen een werkweek na aanvraag beschikbaar.
+
+Binnen de projectomgeving bepaalt het projectteam zelf de gewenste configuratie.
```

Comment:
Is projectomgeving hetzelfde als digitale omgeving? Zo ja, dan graag dezelfde term gebruiken en niet een nieuwe term introduceren.